src/components/ui/Walkthrough.js

```diff
@@ -0,0 +1,74 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+class Walkthrough extends PureComponent {
+
+  static propTypes = {
+    title: PropTypes.string.isRequired,
```

Comment:
Ok sure that makes sense. As we are close to closing the features not going to cause and confusion 👍